### ISSUE TITLE
Feature request: Transparent Key Handling

### ISSUE THREAD

**Issue by randombit** [OpenKMIP/PyKMIP#433]
No idea how hard this would be to support but I figure it can't hurt to ask ;)

**Comment by randombit** [comment]
Yes exactly those. (Mostly just RSA really but EC would be useful for us also.)

**Comment by PeterHamilton** [comment]
@randombit Gotcha. For issue tracking purposes, I'll list the RSA and EC ones out here:

- TransparentRSAPrivateKey
- TransparentRSAPublicKey
- TransparentECPrivateKey
- TransparentECPublicKey

It shouldn't be a heavy lift to add these in. I'll add 'em to the queue.
